panoptes_client/workflow.py

```diff
@@ -48,6 +61,10 @@ def save(self):
         """
```

Comment:
Could you update the docstring here now that it's not just configuration that it's handling?